tiiuae/dev-team-sandbox
# GIVC: Add wifi service into systemd server

## Description of changes
Wifi service is created.
GiVC ssh keys are created for netvm connection.
wifi-list query is implemented in wifi service.

Waiting to be merged [ghaf-givc PR#3](https://github.com/tiiuae/ghaf-givc/pull/3)



## Checklist for things done






## Testing

- Connect to host via SSH
- Use grpcurl to query of wifi network list: `grpcurl -cacert /etc/givc/ca.ghaf/ca-cert.pem -cert /etc/givc/ghaf-host.ghaf/ghaf-host.ghaf-cert.pem -key /etc/givc/ghaf-host.ghaf/ghaf-host.ghaf-key.pem 192.168.101.2:9000 systemd.WifiService.MonitorWifiNetworks`
- Observe the network list with in-use, signal strength and security columns.
